### PR TITLE
Make testsuite windows compatible

### PR DESCRIPTION
**Pull request by StarpTech**
First attempt to fix issues to run tests on windows

Outstanding issues:

- [ ] Absolute paths produced by `vfile-reporter-pretty`
- [ ] Some tests will fail with completely different errors on windows for instance:

```bash
should fail when writing to non-existent dirs
not ok 94 should report
  ---
    operator: deepEqual
    expected: |-
      [ null, 1, 'one.txt\n  1:1  error  Error: Cannot read output directory. Error:' ]
    actual: |-
      [ null, 1, 'one.txt\n  1:1  error  Error: Cannot write multiple files to single output: D:\\Repositories\\unified-engine\\test\\fixtures\\two-files\\three' ]
    at: onrun (D:\Repositories\unified-engine\test\output.js:471:10)
    stack: |-
```

### CODE REVIEW

File: test/util/figures.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+'use strict'
+
+var isWin = require('./isWin')
+
+module.exports = Object.assign(getFigure, figures)
```
Comment (ChristianMurphy) [inline test/util/figures.js]:
```
 ✖  5:43  figures was used before it was defined.                 no-use-before-define
```

File: test/util/isWin.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+'use strict'
```
Comment (ChristianMurphy) [inline test/util/isWin.js]:
```
✖  1:1   Filename is not in kebab case. Rename it to is-win.js.  unicorn/filename-case
```

File: test/color.js [hunk 3]
```diff
@@ -4,6 +4,7 @@ var path = require('path')
 var test = require('tape')
 var noop = require('./util/noop-processor')
 var spy = require('./util/spy')
+var figures = require('./util/figures')
```
Comment (wooorm) [inline test/color.js]: Would it make sense to depend on [`log-symbols`](https://github.com/sindresorhus/eslint-formatter-pretty/blob/df5362c1259fd5d5a7a52cfb196321a54147efa5/index.js#L4) directly? That would guarantee cross-platform workings I think?

File: test/util/isWin.js (hunk 2 above)
Comment (wooorm) [reply test/util/isWin.js]: And [`log-symbols`](https://github.com/sindresorhus/log-symbols/blob/8b40bbb50575f70ec2eec4674f943b266736e372/index.js#L4) seems to use this line:

```js
const isSupported = process.platform !== 'win32' || process.env.CI || process.env.TERM === 'xterm-256color';
```

File: test/color.js (hunk 3 above)
Comment (StarpTech) [reply test/color.js]: Yes, this was my thought as well.

Comment (StarpTech) [reply test/color.js]: We can rely on [`figures`](https://github.com/sindresorhus/figures) otherwise color support can't be optin.

**Pull request closed without merging**